lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,74 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
   margin: 0;
-  padding-top: 20px;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-xl $nx-spacing-l;
+  text-align: right;
```

Comment:
There's also the question of how the form and tile styles are supposed to commingle in the case of a tile that contains nothing but a form.  Do you use `.nx-form-footer` or `.nx-tile-footer`?  Or both on the same element?  If the latter, which element is the `<form>`? The whole `.nx-tile`?

Reply:
in the case of a tile containing just a form, it should use the nx-form footer

Reply:
> in the case of a tile containing just a form, it should use the nx-form footer

Could you clarify what that means from a design perspective?  I was thinking this was more of an implementation question.

Reply:
> > in the case of a tile containing just a form, it should use the nx-form footer
> 
> Could you clarify what that means from a design perspective? I was thinking this was more of an implementation question.

One thing that makes this confusing is that the form footer and the tile footer are slightly different and I'm not sure why we would want that. I propose that they should have the same spacing, same top border, both should either use `nx-btn-bar` or not.

Reply:
> There's also the question of how the form and tile styles are supposed to commingle in the case of a tile that contains nothing but a form. Do you use `.nx-form-footer` or `.nx-tile-footer`? Or both on the same element? If the latter, which element is the `<form>`? The whole `.nx-tile`?

We can write `<form className="nx-tile">` and it works aside from the discrepancies between the footers mentioned above. Remember that most tiles have a tile header, which (to me anyway) strongly suggests that we use `nx-tile-footer` rather than `nx-form-footer` for consistency.

i.e. this seems wrong to me:
```
<form className="nx-tile">
  <header className="nx-tileheader">Header</header>
  <div className="nx-tile-content">Form fields etc</div>
  <footer className="nx-form-footer"><div className="nx-btn-bar">Buttons</div></footer>
</form>
```

Reply:
> One thing that makes this confusing is that the form footer and the tile footer are slightly different and I'm not sure why we would want that. I propose that they should have the same spacing, same top border, both should either use `nx-btn-bar` or not.

both form and tile footers should use nx-btn bar going forward. To Glen's point, this should correct the inconsistencies between nx-tile footer and nx-form footer

Reply:
> > One thing that makes this confusing is that the form footer and the tile footer are slightly different and I'm not sure why we would want that. I propose that they should have the same spacing, same top border, both should either use `nx-btn-bar` or not.
> 
> both form and tile footers should use nx-btn bar going forward. To Glen's point, this should correct the inconsistencies between nx-tile footer and nx-form footer

It does not clear up the inconsistencies though:
```
.nx-form-footer {
  @include container-vertical;
  border-top: 1px solid #e9e9e9;
  padding: $nx-spacing-l;
```

vs

```
.nx-tile-footer {
  @include container-horizontal;
  border-top: 1px solid #e9e9e9;
  margin-top: $nx-spacing-md;
  padding: $nx-spacing-xl 0;
  text-align: right;
}
```

The primary concern is the different padding. The other discrepancies either make sense or are trivial to excise.

Reply:
I created https://issues.sonatype.org/browse/RSC-253 to handle the changes to the tiles.